Make.config

```diff
@@ -98,18 +98,20 @@ MIN_OSX_BUILD_VERSION=10.14
 MIN_OSX_VERSION_FOR_IOS=10.11
 MIN_OSX_VERSION_FOR_MAC=10.11
 
+# WARNING: Do **not** use versions higher than the available Xcode SDK or else we will have issues with mtouch (See https://github.com/xamarin/xamarin-macios/issues/7705)
 IOS_SDK_VERSION=13.2
 # When bumping OSX_SDK_VERSION also update the macOS version where we execute on bots in jenkins/Jenkinsfile (in the 'node' element)
 OSX_SDK_VERSION=10.15
 WATCH_SDK_VERSION=6.1
 TVOS_SDK_VERSION=13.2
 
-# If the max OS version does not match the SDK version (for instance the iOS
-# 12.2 SDK supported both iOS 12.3 and iOS 12.4), then these variables can be
-# set to something other than the corresponding SDK versions.
-MAX_IOS_VERSION=13.3
-MAX_WATCH_VERSION=$(WATCH_SDK_VERSION)
-MAX_TVOS_VERSION=13.3
+MAX_IOS_DEPLOYMENT_TARGET=13.2
```

Comment:
This should be 13.3, since that's what Xcode 11.3 actually supports.

Reply:
Then we should have DEPLOYMENT and SDK, right?

Reply:
Ignore me, you are right.